bundle/deploy/lock/release.go

```diff
@@ -45,10 +47,21 @@ func (m *release) Apply(ctx context.Context, b *bundle.Bundle) error {
 	log.Infof(ctx, "Releasing deployment lock")
 	switch m.goal {
 	case GoalDeploy:
-		return b.Locker.Unlock(ctx)
+		err := b.Locker.Unlock(ctx)
+		if err != nil {
+			return err
+		}
 	case GoalDestroy:
-		return b.Locker.Unlock(ctx, locker.AllowLockFileNotExist)
+		err := b.Locker.Unlock(ctx, locker.AllowLockFileNotExist)
+		if err != nil {
+			return err
+		}
 	default:
 		return fmt.Errorf("unknown goal for lock release: %s", m.goal)
 	}
+
+	if m.successMessage != "" && bundle.ErrFromContext(ctx) == nil {
+		cmdio.LogString(ctx, m.successMessage)
+	}
```

Comment:
I like how putting the main error in the context allows us to display a success message (or failure message, even) when everything is done. The current approach it piggybacks onto the lock release mutator. Could you move it to its own mutator to print a success message if the context err is nil? Then it can be combined with the lock release in a `bundle.Seq` on the finally block.

Reply:
Sure I can look into that.

Reply:
Updated!